tests/functional/service/S3Monitor.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ *
+ *
+ */
+
+require_once '/mnt/test/vendor/autoload.php';
+
+use Aws\Sts\StsClient;
+use Aws\Exception\AwsException;
+use Aws\S3\S3Client;
+
+$seenKeys = [];
+
+/**
+ * Assume Role
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+try {
+    $client = new StsClient([
+        'region' => 'eu-west-1',
+        'version' => 'latest'
+    ]);
+
+    if (getenv('CI')) {
+        $roleToAssumeArn = 'arn:aws:iam::050256574573:role/ci';
+    }
```

Comment:
This role needs to be replaced with the new ci role for lpa

Reply:
ignored for now